clients/client-java/src/main/java/com/datastrato/gravitino/client/ErrorHandlers.java

```diff
@@ -350,21 +355,20 @@ public ErrorResponse parseResponse(int code, String json, ObjectMapper mapper) {
      */
     @Override
     public void accept(ErrorResponse errorResponse) {
-      if (errorResponse.getType() != null) {
```

Comment:
Is this null check still needed?

Reply:
@justinmclean I understand that this null check is no longer needed because there are only two cases in which super.accept(xxx) can be executed:
1. errorResponse.getType() returns a null value
2.errorResponse.getType () returns a non-null value, but it is not one of the type constants defined in OAuth2ClientUtil

The default clause covers both cases. 
cc @yuqi1129

Reply:
AFAIK, switching a null value will cause NPE.

Reply:
> AFAIK, switching a null value will cause NPE.

Got it. Let me test it

Reply:
![image](https://github.com/datastrato/gravitino/assets/94670132/ddd64f40-6574-4048-8a64-b1a2717d1f4b)
Hi @yuqi1129 , thanks for pointing out my mistake, and I have tested that `NPE` did occur when switching null value.

![image](https://github.com/datastrato/gravitino/assets/94670132/a3bbe07f-0e72-4b5a-852d-22ad1b953e1d)
I checked the compiled class file and found that the switch statement needed to call the `hashcode` method of the arguments passed in.  Therefore switching null value will cause `NPE`.

cc @jerryshao @justinmclean

Reply:
Fixed it.